clients/web/src/templates/layout/layoutFooter.pug

```diff
@@ -1,6 +1,7 @@
 .g-footer-links
   a#g-about-link(href="http://girder.readthedocs.org/en/latest/user-guide.html") About
-  a#g-contact-link(href="mailto:kitware@kitware.com") Contact
+  if (contactHref !== '')
```

Comment:
[Pug conditionals](https://pugjs.org/language/conditionals.html) don't need parentheses.

I think you can check whether this is truthy at all (`if contactHref`), which will fail if it's `null` (intentionally not set) or an empty string (intentionally set, but empty).